locale/en/manager.po

```diff
@@ -1238,21 +1238,37 @@ msgstr "Never Remind"
 msgid "manager.setup.reviewOptions.noteOnModification"
 msgstr "Defaults can be modified for each review during the editorial process."
 
-msgid "manager.setup.reviewOptions.reminders.response"
-msgstr "Response Reminder"
+msgid "manager.setup.reviewOptions.reminders"
+msgstr "Set Reminders for Review"
 
-msgid "manager.setup.reviewOptions.reminders.response.description"
+msgid "manager.setup.reviewOptions.reminders.description"
 msgstr ""
-"Send an email reminder if a reviewer has not responded to a review request "
-"this many days after the response due date."
+"Send an email reminder before or after for review request response(if reviewer has not responded "
+"to review request yet) or review submission (if reviewer has not submitted review yet)"
 
-msgid "manager.setup.reviewOptions.reminders.submit"
-msgstr "Review Reminder"
+msgid "manager.setup.reviewOptions.reminders.response.before"
+msgstr "Response Request Response - Before Due Date"
```

Comment:
Should be "Review Request Response", not "Response Request Response"

Reply:
fixed.